gui/src/components/DirectoryBrowser.svelte

```diff
@@ -121,6 +121,6 @@
 
   button:focus,
   button:focus-within {
-    background: var(--grey-d-color);
+    box-shadow: var(--shadow-focus) !important;
```

Comment:
important? ಠ_ಠ 

:)

Reply:
Otherwise I'd need to put `:not(:focus):not(:focus-within)` on the hover style